ruby/person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, _parent_permission, name = 'Unknownn')
```

Comment:
- [ ] Please edit the `parent_permission` parameter to be true as default like so: `parent_permission = true`